geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AvailableTables.scala

```diff
@@ -9,11 +9,12 @@
 package org.locationtech.geomesa.accumulo.data.tables
 
 object AvailableTables {
```

Comment:
good thinking. I guess there could be schemas where we didn't write the key and I bet then it would use that. i'll look